dbt_project.yml

```diff
@@ -469,6 +469,13 @@ models:
         +schema: hashflow_ethereum
         +materialized: view
 
+    nomad:
+      +schema: nexusmutual
+      +materialized: view
+      ethereum:
+        +schema: nexusmutual_ethereum
+        +materialized: view
+
```

Comment:
You may mistakenly name the model as `nomad`, which is another existing model. I will commit a fix.